maven-model-builder/src/main/java/org/apache/maven/model/validation/DefaultModelValidator.java

```diff
@@ -1163,6 +1247,27 @@ private boolean validateVersion( String fieldName, ModelProblemCollector problem
 
     }
 
+    @SuppressWarnings( "checkstyle:parameternumber" )
+    private boolean validateVersion( String prefix, String fieldName, ModelProblemCollector problems, Severity severity,
+                                     Version version, String string, String sourceHint, InputLocationTracker tracker )
+    {
+        if ( string == null || string.length() <= 0 )
```

Comment:
`StringUtils#isEmpty()`?

Reply:
This isn't new code, I'll push a new commit that modifies the existing method instead of duplicating it.